pages/common/gradle.md

```diff
@@ -11,15 +11,15 @@
 
 `gradle build -x {{test}}`
 
-- Run in offline mode to prevent gradle from accessing the network during builds:
+- Run in offline mode to prevent Gradle from accessing the network during builds:
 
 `gradle build --offline`
 
 - Clear the build directory:
 
 `gradle clean`
 
-- Compile and Release package:
+- Build a release `Android Package Kit` package (specific to `Android Studio):
```

Comment:
Not sur I should use backticks here... 🤔

Reply:
I'm not sure about expand `APK` because it's harder to associate that you're talking about an `APK` file. Also, it isn't specific for `Android Studio` because you can use this independently of it, it's used with `react-native` too.
```suggestion
- Build an APK in release mode:
```

Reply:
Do you mean that `gradle assembleRelease` (not just APK format) also works for `react-native` projects? Any resources?

Reply:
You can name the Gradle tasks as you want, but you don't need Android Studio installed to run the `gradle assembleRelease`, you need the Android SDK, and you can install it with the `sdkmanager` (used in CI jobs)
the `gradle assembleRelease` is used to build the Android APK for RN. The docs talk about the `bundleRelease` for the `.aab` file, but the another task exists.

Reply:
@nicokosi Did you read this comment ? :point_up: